e2e/tests/noticeOfChange_test.js

```diff
@@ -1,29 +1,51 @@
 const config = require('../config.js');
+const apiHelper = require('../helpers/api_helper.js');
 const mandatorySubmissionFields = require('../fixtures/caseData/mandatorySubmissionFields.json');
+const representedCase = require('../fixtures/caseData/representedCase.json');
 
 let caseId;
+let privateSolicitorDetails;
 
 Feature('Notice of change');
 
-BeforeSuite(async ({I}) => {
-  caseId = await I.submitNewCaseWithData(mandatorySubmissionFields);
+BeforeSuite(async () => {
+  privateSolicitorDetails = await apiHelper.getUser(config.privateSolicitorOne);
 });
 
-Scenario('Private solicitor obtains case access through NoC', async ({I, caseListPage, noticeOfChangePage}) => {
+Scenario('Private solicitor obtains access to an unrepresented case', async ({I, caseListPage, caseViewPage, noticeOfChangePage}) => {
+  caseId = await I.submitNewCaseWithData(mandatorySubmissionFields);
   await I.signIn(config.privateSolicitorOne);
   I.navigateToCaseList();
   caseListPage.searchForCasesWithId(caseId);
   I.dontSeeCaseInSearchResult(caseId);
-  noticeOfChangePage.navigate();
-  await noticeOfChangePage.enterCaseReference(caseId);
-  await I.retryUntilExists(() => I.click('Continue'), noticeOfChangePage.fields.applicantName);
-  await noticeOfChangePage.enterApplicantName('Swansea City Council');
-  noticeOfChangePage.enterRespondentName('Joe', 'Bloggs');
-  await I.retryUntilExists(() => I.click('Continue'), noticeOfChangePage.fields.confirmNoC);
-  noticeOfChangePage.confirmNoticeOfChange();
-  await I.retryUntilExists(() => I.click('Submit'), '.govuk-panel--confirmation');
-  I.see('Notice of change successful');
+
+  await noticeOfChangePage.userCompletesNoC(caseId, 'Swansea City Council', 'Joe', 'Bloggs');
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertRepresentative(I, privateSolicitorDetails.forename, privateSolicitorDetails.surname, privateSolicitorDetails.email, 'Private solicitors');
```

Comment:
the test organisation data in preview and AAT is `"FPL Private Solicitors"`, so this will prob need to be
```suggestion
  assertRepresentative(I, privateSolicitorDetails.forename, privateSolicitorDetails.surname, privateSolicitorDetails.email, 'FPL Private Solicitors');
```

Reply:
AAT should now read 'Private solicitors'